<launch>

<node pkg="eklavya4_roboteq" type="roboteq_server" name="roboteq_server" respawn="true" output="screen" />
<node pkg="eklavya4_roboteq" type="roboteq_client" name="roboteq_client" respawn="true" output="screen" />

<node pkg="eklavya4_roboteq" type="joy2" name="joy2" respawn="true" output="screen" />
<node pkg="joy" type="joy_node" name="joy_node" respawn="true" output="screen" />

</launch>
